_includes/field-description-description.md

```diff
@@ -0,0 +1,14 @@
+{%- capture description -%}
+    A 40 character length textual [description][description] of the purchase.
```

Comment:
We need the footnote link for `[description]` in the include and probably a `payment_instrument` parameter so the link can be constructed relative to the payment instrument.